examples/apps/quick_tap.py

```diff
@@ -0,0 +1,415 @@
+        if self.buzzer_display_type == BUZZER_RED:
+            self.turn_on_buzzer_cubes_red()
+        elif self.buzzer_display_type == BUZZER_DIFFERENT:
+            self.turn_on_buzzer_cubes_different()
+        elif self.buzzer_display_type == BUZZER_SAME:
+            self.turn_on_buzzer_cubes_same()
+
+    def turn_on_buzzer_cubes_same(self):
+        '''Sets the buzzer cubes to the same randomly generated color pair.'''
+        same_colors = self.generate_random_buzzer_colors()
+        self.player.cube.set_light_corners(*same_colors)
+        self.cozmo_player.cube.set_light_corners(*same_colors)
+
+    def turn_on_buzzer_cubes_different(self):
+        '''Sets the buzzer cubes to different randomly generated color pairs.'''
+        player_cube_colors = self.generate_random_buzzer_colors()
+        cozmo_cube_colors = self.generate_random_buzzer_colors()
+        while player_cube_colors == cozmo_cube_colors:
+            cozmo_cube_colors = self.generate_random_buzzer_colors()
+        self.player.cube.set_light_corners(*player_cube_colors)
+        self.cozmo_player.cube.set_light_corners(*cozmo_cube_colors)
+
+    def turn_on_buzzer_cubes_red(self):
+        '''Sets the buzzer cubes to red.'''
+        self.player.cube.set_lights(cozmo.lights.red_light)
+        self.cozmo_player.cube.set_lights(cozmo.lights.red_light)
+
+    def generate_random_buzzer_colors(self):
+        '''Creates a list of different alternating colors, chosen randomly from LIGHT_COLORS_LIST.
+
+        Returns:
+            a list of Lights from LIGHT_COLORS_LIST'''
+        num_colors = len(LIGHT_COLORS_LIST)
+        x = random.randrange(num_colors)
+        y = random.randrange(num_colors)
+        while y == x:
+            y = random.randrange(num_colors)
+        return [LIGHT_COLORS_LIST[x], LIGHT_COLORS_LIST[y], LIGHT_COLORS_LIST[x], LIGHT_COLORS_LIST[y]]
+
+    def turn_off_buzzer_cubes(self):
+        '''Turns off both buzzer cubes' lights.'''
+        self.player.cube.set_lights_off()
+        self.cozmo_player.cube.set_lights_off()
+
+    def cubes_connected(self):
+        '''Returns true if Cozmo connects to all three cubes successfully.'''
+        cube1 = self.robot.world.get_light_cube(cozmo.objects.LightCube1Id)
+        cube2 = self.robot.world.get_light_cube(cozmo.objects.LightCube2Id)
+        cube3 = self.robot.world.get_light_cube(cozmo.objects.LightCube3Id)
+        self.cubes = [cube1, cube2, cube3]
+        return not (cube1 == None or cube2 == None or cube3 == None)
+
+    def report_scores(self):
+        '''Prints the current scores of the game.'''
+        print("Player score: {}".format(self.player.score))
+        print("Cozmo score: {}".format(self.cozmo_player.score))
+
+    async def report_winner(self):
+        '''Prints the final scores of the game, and the winner.'''
+        print("---------------------------------------------------")
+        print("You won {} rounds".format(self.player.score))
```

Comment:
Could make the text smarter to print `round` if either Cozmo or the player only won 1 round.